### PR TITLE
Change tabs to spaces in echo messages for build.xml files

### PR DESCRIPTION
**Pull request by sharon-wang**
"echo" messages in the compile target for build.xml should contain spaces instead of tabs. 
These messages in build.xml files in the `test` directory have been updated to contain only spaces.

Signed-off-by: Sharon Wang <sharon-wang-cpsc@outlook.com>

### CODE REVIEW

**Comment by sharon-wang** [comment]
@keithc-ca PR created for switching tabs to spaces in build.xml files

File: test/functional/cmdline_options_tester/build.xml [hunk 1]
```diff
@@ -45,11 +45,10 @@
 
 	<target name="compile" depends="init" description="compile the source ">
 		<echo>Ant version is ${ant.version}</echo>
-		<echo>============COMPILER SETTINGS============</echo>
```
Comment (keithc-ca) [inline test/functional/cmdline_options_tester/build.xml]: Why was this line removed?

Comment (sharon-wang) [reply test/functional/cmdline_options_tester/build.xml]: Missed this deletion in my local diff - fixing!

**Comment by sharon-wang** [comment]
@keithc-ca Reverted the accidental line deletion - this should be good to go now.

**Comment by keithc-ca** [comment]
Jenkins test sanity plinux jdk11
